Documentation/DocuBlocks/Rest/Database/get_api_database_new.md

```diff
@@ -35,20 +35,20 @@ If *users* is not specified or does not contain any users, a default user
 new database will be accessible after it is created.
 Each user object can contain the following attributes:
 
-@RESTSTRUCT{username,get_api_database_new_USERS,string,required,string}
-Login name of the user to be created
+@RESTSTRUCT{username,get_api_database_new_USERS,string,required,}
+Login name of the user to be created.
 
-@RESTSTRUCT{passwd,get_api_database_new_USERS,string,required,string}
+@RESTSTRUCT{passwd,get_api_database_new_USERS,string,optional,password}
 The user password as a string. If not specified, it will default to an empty string.
 
-@RESTSTRUCT{active,get_api_database_new_USERS,boolean,required,}
+@RESTSTRUCT{active,get_api_database_new_USERS,boolean,optional,}
 A flag indicating whether the user account should be activated or not.
 The default value is *true*. If set to *false*, the user won't be able to
-log into the database.
+log into the database. The default is *true*.
 
 @RESTSTRUCT{extra,get_api_database_new_USERS,object,optional,}
-A JSON object with extra user information. The data contained in *extra*
-will be stored for the user but not be interpreted further by ArangoDB.
+A JSON object with extra user information. It is used by the web interface
+to store graph viewer settings and saved queries.
```

Comment:
we should probably note here that this attribute is not expected to be set/modified by end users.

Reply:
Will do that, but we should also remove this parameter entirely if it's only for internal use (4.0).